addons/point_of_sale/models/pos_order.py

```diff
@@ -1227,3 +1228,17 @@ def _get_report_values(self, docids, data=None):
         configs = self.env['pos.config'].browse(data['config_ids'])
         data.update(self.get_sale_details(data['date_start'], data['date_stop'], configs))
         return data
+
+
+class AccountMoveLine(models.Model):
+    _inherit = 'account.move.line'
+
+    def _stock_account_get_anglo_saxon_price_unit(self):
+        self.ensure_one()
+        if not self.product_id:
+            return self.price_unit
+        price_unit = super(AccountMoveLine, self)._stock_account_get_anglo_saxon_price_unit()
+        order = self.env['pos.order'].search([('account_move', '=', self.move_id.id)])
```

Comment:
so why is this code needed now and not before? was it lost during a refactoring? can you link the commit who broke it?

Reply:
Is the commit message better now?  Everything else is done.

Reply:
maybe create the inverse one2many of pos.order account_move_id to access it more easily

Reply:
and move all of this in a new file named account_move.py containing the inherit on account_move adding the o2m the inherit on account move line containing this _stock_account_get_anglo_saxon_price_unit

Reply:
done